torch/csrc/jit/tensorexpr/half_support.h

```diff
@@ -13,7 +13,7 @@ namespace tensorexpr {
 class HalfChecker : public IRVisitor {
  public:
   HalfChecker(const std::vector<CodeGen::BufferArg>& args) {
-    for (const auto& BA : args) {
```

Comment:
I think we can keep `const` in `const auto&` - it probably doesn't create any issues with constness.

Reply:
Done.